tests/test_unittest.py

```diff
@@ -18,9 +21,8 @@ def test_failed_test_case(self):
 
 if __name__ == "__main__":
     EXPECTED_FAILURES = 1
-    EXPECTED_TESTS = 1
     test = unittest.main(exit=False)
 
-    if len(test.result.failures) == EXPECTED_FAILURES and test.result.testsRun == EXPECTED_TESTS:
+    if len(test.result.failures) == EXPECTED_FAILURES and test.result.testsRun > EXPECTED_FAILURES:
```

Comment:
(minor nitpick)
```suggestion
    if len(test.result.failures) == EXPECTED_FAILURES and test.result.testsRun >= EXPECTED_FAILURES:
```

Reply:
`test.result.testsRun` should not be equal to `EXPECTED_FAILURES`. It should always be higher. Otherwise this would pass if the inherited test cases would not be imported.

Reply:
Ah all right! Can you put this as a comment there ?

Reply:
Added comment!